src/docs/sphinx/quickstart_guide/config_build.rst

```diff
@@ -276,76 +307,151 @@ directly to CMake; for example::
-|                              |         |                                        |
-|                              |         | By DEFAULT, it is enabled for          |
-|                              |         | `Debug` and `RelWithDebInfo` configs   |
-|                              |         | but this can be overridden by setting  |
-|                              |         | `AXOM_DEBUG_DEFINE` to `ON` or `OFF`   |
-+------------------------------+---------+----------------------------------------+
 | AXOM_ENABLE_ALL_COMPONENTS   | ON      | Enable all components by default       |
 +------------------------------+---------+----------------------------------------+
-| AXOM_ENABLE_<FOO>            | ON      | Enables the axom component named 'foo' |
+| AXOM_ENABLE_<FOO>            | ON      | Enable the axom component named 'Foo'  |
 |                              |         |                                        |
 |                              |         | (e.g. AXOM_ENABLE_SIDRE)               |
-|                              |         | for the sidre component                |
+|                              |         | for the Sidre component                |
 +------------------------------+---------+----------------------------------------+
-| AXOM_ENABLE_DOCS             | ON      | Builds documentation                   |
+| AXOM_ENABLE_EXAMPLES         | ON      | Build Axom examples                    |
 +------------------------------+---------+----------------------------------------+
-| AXOM_ENABLE_EXAMPLES         | ON      | Builds examples                        |
+| AXOM_ENABLE_TESTS            | ON      | Build Axom unit tests                  |
 +------------------------------+---------+----------------------------------------+
-| AXOM_ENABLE_TESTS            | ON      | Builds unit tests                      |
+| ENABLE_BENCHMARKS            | OFF     | Enable Google benchmark                |
 +------------------------------+---------+----------------------------------------+
-| AXOM_ENABLE_TOOLS            | ON      | Builds tools                           |
+| AXOM_ENABLE_DOCS             | ON      | Enable Axom documentation to be built  |
+|                              |         | as a make target                       |
 +------------------------------+---------+----------------------------------------+
-| BUILD_SHARED_LIBS            | OFF     | Build shared libraries.                |
-|                              |         | Default is Static libraries            |
+
+If ``AXOM_ENABLE_ALL_COMPONENTS`` is OFF, you must explicitly enable the desired
+components (other than 'core', which is always enabled).
+
+See `Axom software documentation <../../../index.html>`_
+for a list of Axom components and their dependencies.
+
+See :ref:`dependencies-label` for configuration variables to specify paths
+to Axom external dependencies.
```

Comment:
Suggestion: It might be worth adding that these take the form of a CMake variable `FOO_DIR` with a path to a valid installation for package `Foo`.

Reply:
that point is mentioned a couple of times in the section referenced. I will reinforce the point here.